### ISSUE TITLE
New license request: CC-BY-NC-ND-3.0-IGO [SPDX-Online-Tools]

### ISSUE THREAD

**Issue by alanorth** [spdx/license-list-XML#767]
**1.** License Name: Creative Commons 3.0 Intergovernmental Organizations
**2.** Short identifier: CC-BY-NC-ND-3.0-IGO
**3.** URL: https://wiki.creativecommons.org/wiki/Intergovernmental_Organizations
**4.** OSI Status: Not Submitted

**Comment by DennisClark** [comment]
Please clarify the actual license text of the license being requested.  I do not see any license text at https://wiki.creativecommons.org/wiki/Intergovernmental_Organizations

**Comment by alanorth** [comment]
Thanks @DennisClark. The IGO licenses are a class of Creative Commons license with NC/ND/SA variations just like the standard Creative Commons licenses, so I just gave the link to the general IGO license information page. The license test for CC-BY-NC-ND-3.0-IGO license would be here:

https://creativecommons.org/licenses/by-nc-nd/3.0/igo/

Should I make a request for each of the variations?

**Comment by DennisClark** [comment]
@alanorth   I believe we can start with this one.  I'm afraid that the serious restrictions in the license could be a concern, and may prevent it from meeting SPDX License List guidelines:

```
Attribution — You must give appropriate credit, provide a link to the license, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.

NonCommercial — You may not use the material for commercial purposes.

NoDerivatives — If you remix, transform, or build upon the material, you may not distribute the modified material. 
```

although I suppose that since we are talking about usage by government organizations that other considerations may apply.

**Comment by alanorth** [comment]
@DennisClark Sorry, I don't understand. You mean that SPDX would have a problem with the attribution, non-commercial, and no-derivatives restrictions of the Creative Commons license in general or? The [SPDX list](https://spdx.org/licenses/) already has versions 1.0, 2.0, 2.5, 3.0, and 4.0 of the standard Creative Commons licenses with the exact same restrictions, ie `CC-BY-NC-ND-4.0`.

**Comment by jlovejoy** [comment]
see comments in #763 - it's essentially the same theme.

**Comment by alanorth** [comment]
@jlovejoy thank you for the background information. Looking forward to feedback after the legal call.

**Comment by mlinksva** [comment]
A different issue that this intersects with is that there are literally hundreds of CC licenses "ported" to various jurisdictions not cataloged by SPDX https://github.com/spdx/license-list-XML/issues/618#issuecomment-372803130

It seems like each (non-open licenses, "ported" CC licenses) ought be considered carefully. This license happens to touch both.

**Comment by swinslow** [comment]
Discussed on 2019-03-07 legal team call. Participants had varying opinions, and there was not a clear consensus, whether this one should be added. Observations included the following:

1. CC licenses currently on the License List were initially added prior to the current candidate license guidelines becoming more crystallized. 
2. The fact that other CC-NC-* licenses were previously added is a relevant consideration, but not dispositive, for whether new ones should be added going forward.
3. There is evidence that CC-BY-NC-ND-3.0-IGO has some significant usage in the wild. See 
https://wiki.creativecommons.org/wiki/Intergovernmental_Organizations#Examples_of_CC_License_Use_by_Intergovernmental_Organizations (note though that not all examples listed are this -IGO variant).
4. Whether or not this one is added, there is not a current appetite for adding every single possible other ported CC license.

Anyone who was attending, please weigh in if you feel I'm misrepresenting any of the discussion  :)

The next steps that were discussed included that there may be discussion of this during the SPDX face-to-face next week, but in any event this will be raised again on a subsequent legal team call for further consideration and a decision.

**Comment by jlovejoy** [comment]
discussed again on 2019-03-21 call - still struggling with what to do here. In addition to points above:
- given the status and recognizability of CC as a license steward, it would be good to have the more "blessed" licenses. the IGO variation seems to be such a case (as well as other IGO variants like SA, etc.) but would be good to have input from CC on this...
- this also touches upon bigger issue of accepting more licenses (generally and/or that are not strictly open source) on the SPDX License List, which then in turn touches upon the issue of a small number of volunteers doing all the work.
thus - tagging for next release to allow more time for some of these bigger issue conversations.
@mlinksva  - are you still working with CC?

**Comment by swinslow** [comment]
Circling back here -- I think the "yes or no" on this one is tied to the ongoing discussions for the License Inclusion Guidelines updates being discussed at #925. That is intended to be addressed during 3.8, so I'm moving this to 3.8 also. After a decision is reached on updating the guidelines, we should be able to give this a "yes or no" outcome.

**Comment by swinslow** [comment]
Following up on this request: The SPDX license list inclusion principles have now been updated, see https://github.com/spdx/license-list-XML/blob/master/DOCS/license-inclusion-principles.md.

In light of the update, I would be generally in favor of adding this one to the list. For me the evidence of use in the wild pushes this over the line, particularly given that we've already got other CC-BY-NC licenses on the list.

**Comment by swinslow** [comment]
Discussed on legal team call 2020-04-23, agreed to add this one; will log separate issue for broader discussion of decision-making on other ports of CC licenses.

**Comment by vmbrasseur** [comment]
ohai

**Comment by vmbrasseur** [comment]
Starting initial work on adding this one (first time I'll be adding a license!). I'll be using Jilayne's PR #1030 as an example.

First, though, I should probably go read more documentation. ;-)

**Comment by vmbrasseur** [comment]
Those docs assume the license exists in the License Requests list.

In the case of CC-BY-NC-ND-3.0-IGO, that's not the case.

What's the answer here? Copy/modify the XML for CC-BY-NC-ND-3.0 and modify, or…?

Paging @swinslow to the white courtesy phone.

**Comment by vmbrasseur** [comment]
Discussing on today's call. Surprised this isn't in the license requests list, since it was submitted via the tool.

Turns out it's in the Archived Requests list. Steve unarchived it.

**Comment by swinslow** [comment]
This was merged in #1068. Thank you @vmbrasseur!
